emacs/lsp-cquery.el

```diff
@@ -274,17 +274,22 @@
 (defun cquery//render-string (str)
   (condition-case nil
       (with-temp-buffer
-	(delay-mode-hooks (c++-mode))
-	(insert str)
-	(font-lock-ensure)
-	(buffer-string))
+	    (delay-mode-hooks (c++-mode))
+	    (insert str)
+	    (font-lock-ensure)
+	    (buffer-string))
     (error str)))
 
 (defun cquery//initialize-client (client)
   (dolist (p cquery//handlers)
     (lsp-client-on-notification client (car p) (cdr p)))
   (lsp-provide-marked-string-renderer client "c++" #'cquery//render-string))
 
+(defun cquery//get-init-params (workspace)
```

Comment:
cachedirectory should be a customizable variable, that defaults to the vscode path

Reply:
Done.